### PR TITLE
Bump angular to v12

### PR DESCRIPTION
**Pull request by Hinton**
## Type of change
- [ ] Bug fix
- [ ] New feature development
- [x] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other

## Objective

Bumps Angular to v12, upgrades Webpack to v5.

Depends on: https://github.com/bitwarden/jslib/pull/571

## Before you submit
- [ ] I have checked for **linting** errors (`npm run lint`) (required)
- [ ] This change requires a **documentation update** (notify the documentation team)
- [ ] This change has particular **deployment requirements** (notify the DevOps team)

### CODE REVIEW

File: bitwarden_license/src/app/providers/manage/bulk/bulk-confirm.component.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@ import { BulkConfirmComponent as OrganizationBulkConfirmComponent } from 'src/ap
 import { BulkUserDetails } from 'src/app/organizations/manage/bulk/bulk-status.component';
 
 @Component({
-    templateUrl: '/src/app/organizations/manage/bulk/bulk-confirm.component.html',
+    templateUrl: '../../../../../../src/app/organizations/manage/bulk/bulk-confirm.component.html',
```
Comment (Hinton) [inline bitwarden_license/src/app/providers/manage/bulk/bulk-confirm.component.ts:15]: Need to figure out if there's some way to get the absolute paths working again, this is a pain.

File: webpack.config.js [hunk 2]
```diff
@@ -202,7 +197,7 @@ const devServer = NODE_ENV !== 'development' ? {} : {
         },
     },
     hot: false,
-    allowedHosts: envConfig.dev?.allowedHosts,
+    //allowedHosts: envConfig.dev?.allowedHosts,
```
Comment (Hinton) [inline webpack.config.js:200]: Need to figure out what the new argument s.

**Comment by Hinton** [comment]
Also need to figure out if we need to do further tweaks to the webpack config.

File: bitwarden_license/src/app/providers/manage/bulk/bulk-confirm.component.ts (hunk 1 above)
Comment (Hinton) [reply bitwarden_license/src/app/providers/manage/bulk/bulk-confirm.component.ts:15]: Apparently the old absolute path was never explicitly supported, and was "accidentally fixed in 11.2". And there are no plans to bring back support. Keeping it as is for now. Might be a sign that we should either make the base component more generic, or duplicate the templates.

**Review by djsmith85 — DISMISSED** [review]

**Review by djsmith85 — APPROVED** [review]

**Pull request merged**